MapboxVisionTests/VisionManagerTests.swift

```diff
@@ -61,4 +61,62 @@ class VisionManagerTests: XCTestCase {
             "VisionManager should be successfully deallocated after calling destroy and releasing the object"
         )
     }
+
+    func testPerformanceConfigMergedTranslatesToSamePerformance() {
```

Comment:
nit:

```suggestion
    func testPerformanceConfigMergedTranslatesToTheSamePerformance() {
```

?

Reply:
I wanted to omit all linguistic load that doesn't affect understanding since the function name is huge even without it. WDYT?